### PR TITLE
Windows docker

### PR DESCRIPTION
**Pull request by tabuckner**
## What I Did
Added a shell script for windows users. Tested on @mccormickc 's new laptop while setting up a one click UX Writer script on her desktop.

## How To Test It
Should work if you're on windows and meet the requirements.

### CODE REVIEW

File: scripts/windows-docker.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# JEKYLL_VERSION=$(sed -E -n 's/^.*gem "jekyll", "[<>=~]{1,2} (.*)".*$/\1/p' ./Gemfile)
+JEKYLL_VERSION=3.8.5 # TODO: See if sed will work on windows.
```
Comment (tabuckner) [inline scripts/windows-docker.sh]: @jnewton03 maybe you know what's up here? I didnt actually test SED and had already handed the laptop over when i realized it :/

File: scripts/docker-dev.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
```
Comment (jnewton03) [inline scripts/docker-dev.sh]: use #!/usr/bin/env bash for portability in the shebang

File: scripts/docker-dev.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
 
 JEKYLL_VERSION=$(sed -E -n 's/^.*gem "jekyll", "[<>=~]{1,2} (.*)".*$/\1/p' ./Gemfile)
-docker run --volume="$PWD:/srv/jekyll" -p 3000:4000 -it jekyll/jekyll:$JEKYLL_VERSION jekyll serve --watch --drafts
\ No newline at end of file
+docker run --volume="$PWD:/srv/jekyll" -p 4000:4000 -it jekyll/jekyll:$JEKYLL_VERSION jekyll serve --watch --drafts
```
Comment (jnewton03) [inline scripts/docker-dev.sh]: IIRC you need a / before PWD to support git-bash

Comment (tabuckner) [reply scripts/docker-dev.sh]: this is the mac version of the single startup script, I should rename it to make it more clear.

**Pull request merged**